integration-cli/docker_cli_events_test.go

```diff
@@ -327,7 +327,7 @@ func (s *DockerSuite) TestEventsFilterContainer(c *check.C) {
 			c.Fatal(err)
 		}
 		nameID[name] = strings.TrimSpace(out)
-		waitInspect(name, "{{.State.Runing }}", "false", 5)
+		waitInspect(name, "{{.State.Running }}", "false", 5)
```

Comment:
Can we please replace this by a call to [`waitRun`](https://github.com/docker/docker/blob/eae272f90eb6c01824112054140b5dc7d9d679ce/integration-cli/utils.go#L202-L204)?